packages/gestalt/src/Avatar.js

```diff
@@ -60,6 +60,7 @@ type AvatarProps = {|
   size?: 'sm' | 'md' | 'lg',
   src?: string,
   verified?: boolean,
+  icon?: 'check-circle' | 'pinterest',
```

Comment:
One more thing - can we update `Avatar.doc.js` to mention this new prop and to provide an example please